guacamole-docker/bin/build-guacamole.sh

```diff
@@ -53,7 +54,12 @@ mkdir -p "$DESTINATION"
 #
 
 cd "$BUILD_DIR"
-mvn package
+
+if [ -z "$BUILD_PROFILE" ]; then
+  mvn package
+else
+  mvn -P "$BUILD_PROFILE" package
+fi
```

Comment:
Please stick with accepted style of four-space indentations.

Reply:
corrected